src/mame/snk/hng64.cpp

```diff
@@ -1720,20 +1720,47 @@ static const gfx_layout hng64_16x16x8_spritelayout =
 };
 
 static const uint32_t texlayout_xoffset[1024] = { STEP1024(0,8) };
-static const uint32_t texlayout_yoffset[512] = { STEP512(0,8192) };
-static const gfx_layout hng64_texlayout =
+static const uint32_t texlayout_yoffset[1024] = { STEP1024(0,8192) };
+
+static uint32_t texlayout_xoffset_4[1024];
+static const uint32_t texlayout_yoffset_4[1024] = { STEP1024(0,4096) };
+
+void hng64_state::texlayout_xoffset_4_create()
+{
+	for (int i = 0; i < 1024; i++)
+	{
+		texlayout_xoffset_4[i] = (i * 4) ^ 4;
+	}
+}
```

Comment:
It’s actually possible to generate this at compile time.  I’ll patch it up after merging I guess.